scripts/bridge/populate_external_bridge.py

```
@@ -0,0 +1,135 @@
+import psycopg2
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_db_connection():
+    """Get PostgreSQL connection from environment variables."""
+    db_url = os.getenv("DATABASE_URL")
+    if db_url:
+        return psycopg2.connect(db_url)
+    else:
```

Ruff (0.15.10)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)